docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -67,6 +67,7 @@ Property name                              Description
 ``mongodb.write-concern``                  The write concern
 ``mongodb.required-replica-set``           The required replica set name
 ``mongodb.cursor-batch-size``              The number of elements to return in a batch
+``mongodb.projection-pushdown-enabled``    Read only projected fields from row columns while performing ``SELECT`` queries
```

Comment:
Why do we need this config toggle? Why users would like to have this disabled? If it is only a kill switch I would prefer not document it to make it easier to remove it later.

Reply:
Yes, The purpose of the flag is a kill switch.

Reply:
Removed from doc.